mysql/azuremysql: tests are failing

Running via https://github.com/google/go-cloud/blob/master/internal/testing/prereleasechecks.sh, the Terraform steps succeed but then the tests fail:

```
$ ./internal/testing/prereleasechecks.sh run
***** mysql/azuremysql *****
--- FAIL: TestOpen (1.54s)
    azuremysql_test.go:63: Ping:  Error 9999: An internal error has occurred. Please retry or report your issues.
    azuremysql_test.go:66: ExecContext:  Error 9999: An internal error has occurred. Please retry or report your issues.
    azuremysql_test.go:69: ExecContext:  Error 9999: An internal error has occurred. Please retry or report your issues.
FAIL
FAIL	gocloud.dev/mysql/azuremysql	1.611s
```

--- a/internal/testing/prereleasechecks.sh
+++ b/internal/testing/prereleasechecks.sh
@@ -59,8 +59,7 @@ case "$op" in
     terraform init && terraform apply -var location="centralus" -var resourcegroup="GoCloud" -auto-approve || FAILURES="$FAILURES $TESTDIR"
     ;;
   run)
-    # TODO: These tests fail with "Error 9999".
-    go test -mod=readonly -race -json | go run "$rootdir"/internal/testing/test-summary/test-summary.go -progress || echo "[KNOWN FAILURE]"
+    go test -mod=readonly -race -json | go run "$rootdir"/internal/testing/test-summary/test-summary.go -progress || FAILURES="$FAILURES $TESTDIR"
     ;;
   cleanup)
     terraform destroy -var location="centralus" -var resourcegroup="GoCloud" -auto-approve || FAILURES="$FAILURES $TESTDIR"

--- a/mysql/azuremysql/main.tf
+++ b/mysql/azuremysql/main.tf
@@ -73,7 +73,7 @@ resource "azurerm_mysql_firewall_rule" "addrule" {
   resource_group_name = "${azurerm_resource_group.mysqlrg.name}"
   server_name         = "${azurerm_mysql_server.mysqlserver.name}"
   start_ip_address    = "0.0.0.0"
-  end_ip_address      = "0.0.0.0"
+  end_ip_address      = "255.255.255.255"
 }
 
 resource "azurerm_mysql_database" "mysqldb" {
@@ -85,7 +85,7 @@ resource "azurerm_mysql_database" "mysqldb" {
 }
 
 output "username" {
-  value       = "gocloudadmin"
+  value       = "${azurerm_mysql_server.mysqlserver.administrator_login}@${azurerm_mysql_server.mysqlserver.name}"
   description = "The MySQL username to connect with."
 }
 

